helm/chaos-mesh/values.yaml

```diff
@@ -33,7 +33,7 @@ controllerManager:
 
   priorityClassName: ""
 
-  image: pingcap/chaos-mesh:latest
+  image: pingcap/chaos-mesh:v1.2.0
```

Comment:
I can't find v1.2.0 image from the docker hub: https://registry.hub.docker.com/r/pingcap/chaos-mesh/tags?page=1&ordering=last_updated&name=v.1.20, should we upload it first?

Reply:
At present, we update the default tag first and then build the image after this pr merged. Because I want to add this pr to v1.2.0 tag. This may be not the best practice. We will adjust the release process in the future and plan to pull a proposal that our maintainers and committers take turns to be the release manager.